core/src/models/Destination.ts

```diff
@@ -294,12 +294,12 @@ export class Destination extends LoggedModel<Destination> {
     return DestinationOps.exportGroupMembers(this, force);
   }
 
-  async trackGroup(group: Group) {
-    return DestinationOps.trackGroup(this, group);
+  async trackGroup(group: Group, force = true) {
+    return DestinationOps.trackGroup(this, group, force);
   }
 
-  async unTrackGroup() {
-    return DestinationOps.unTrackGroup(this);
+  async unTrackGroup(force = false) {
```

Comment:
Confirming behavior:
- `trackGroup` has a default of `force=true` for the run
- `unTrackGroup` has a default of `force=false` for the run